pkg/applicationmanager/controllers/applicationrestore.go

```diff
@@ -1124,6 +1124,7 @@ func (a *ApplicationRestoreController) skipVolumesFromRestoreList(
 		restoreVolInfo.Status = storkapi.ApplicationRestoreStatusRetained
 		restoreVolInfo.RestoreVolume = pvc.Spec.VolumeName
 		restoreVolInfo.TotalSize = bkupVolInfo.TotalSize
+		restoreVolInfo.Zones = bkupVolInfo.Zones
```

Comment:
Can we test it with restoring to cluster on different zone? I feel, we are overwriting the zone value to source cluster zone.

Reply:
As discussed , modified to fetch zones from PV objects. Implemented the changes for EKS as well